meilisearch-http/src/routes/search.rs

```diff
@@ -28,6 +30,7 @@ struct SearchQuery {
     attributes_to_highlight: Option<String>,
     filters: Option<String>,
     matches: Option<bool>,
+    facet_filters: Option<String>,
```

Comment:
I am not sure about that, wouldn't it be possible to accept an `Vec<Either<Vec<String>, String>>` here?
We would benefit from the serde error messages when parsing the user request.
Is it because query parameter parsing doesn't works correctly with `Either` and `Vec`s?

Reply:
I tried, and unfortunately, it doesn't work... :( would have saved me some work